crypto/rand/drbg_lib.c

```diff
@@ -544,34 +729,42 @@ static int drbg_setup(RAND_DRBG *drbg, const char *name)
  */
 DEFINE_RUN_ONCE_STATIC(do_rand_drbg_init)
 {
-    int ret = 1;
+    drbg_master  = drbg_setup("drbg_master",  NULL);
+    drbg_public  = drbg_setup("drbg_public",  drbg_master);
+    drbg_private = drbg_setup("drbg_private", drbg_master);
```

Comment:
I'm not really a fan of the extra spaces

Reply:
Added a fixup to remove the extra spaces.